src/Service/ContainerAbstractServiceFactory.php

```diff
@@ -60,7 +60,7 @@ class ContainerAbstractServiceFactory implements AbstractFactoryInterface
      * @param  string                  $requestedName
      * @return bool
      */
-    public function canCreateServiceWithName(ContainerInterface $container, $requestedName)
+    public function canCreate(ContainerInterface $container, $requestedName)
```

Comment:
Both are necessary. `canCreateServiceWithName()` is for usage with zend-servicemanager v2, and should proxy to `canCreate()`. I'll make that change on merge.